pkg/lib/ownerutil/util.go

```diff
@@ -394,10 +395,10 @@ func InferGroupVersionKind(obj runtime.Object) error {
 			Version: operatorsv1.GroupVersion.Version,
 			Kind:    "OperatorGroup",
 		})
-	case *operatorsv1.OperatorCondition:
+	case *operatorsv2.OperatorCondition:
```

Comment:
I believe this change is fine, but wanted to invite others to consider possible scenarios where OLM would be unable to infer the GVK for the v1 operator condition. I don't believe it is possible to OLM to hit this scenario because all getters are at v2.

Reply:
We can keep v1 here and simply add v2 if that's what we prefer.